e2e/cypress/integration/accessibility/accessibility_dropdowns_spec.js

```diff
@@ -89,81 +84,69 @@ describe('Verify Accessibility Support in Dropdown Menus', () => {
     });
 
     it('MM-T1476 Accessibility Support in Main Menu Dropdown', () => {
-        // # Press tab from the Set Status button
-        cy.get('.status-wrapper button.status').focus().wait(TIMEOUTS.HALF_SEC).tab({shift: true}).tab().tab();
-
-        // * Verify the aria-label in main menu button
-        cy.get('#headerInfo button').should('have.attr', 'aria-label', 'main menu').and('have.class', 'a11y--active a11y--focused').click();
+        // # Open team menu
+        cy.uiGetLHSHeader().click();
 
         // * Verify the accessibility support in the Main Menu Dropdown
-        cy.get('#sidebarDropdownMenu').should('have.attr', 'aria-label', 'main menu').and('have.class', 'a11y__popup').and('have.attr', 'role', 'menu');
+        cy.findByRole('menu').
+            should('exist').
+            and('have.attr', 'aria-label', 'main menu').
+            and('have.class', 'a11y__popup');
 
         // * Verify the first option is not selected by default
-        cy.get('#sidebarDropdownMenu .MenuItem').children().eq(0).should('not.have.class', 'a11y--active a11y--focused');
-
-        // # Press tab
-        cy.focused().tab();
+        cy.uiGetLHSTeamMenu().find('.MenuItem').
+            children().eq(0).
+            should('not.have.class', 'a11y--active a11y--focused').
+            focus();
 
         // * Verify the accessibility support in the Main Menu Dropdown items
         const menuItems = [
-            {id: 'accountSettings', label: 'Account Settings dialog'},
             {id: 'invitePeople', label: 'Invite People dialog'},
             {id: 'teamSettings', label: 'Team Settings dialog'},
             {id: 'manageMembers', label: 'Manage Members dialog'},
-            {id: 'createTeam', text: 'Create a Team'},
             {id: 'joinTeam', text: 'Join Another Team'},
             {id: 'leaveTeam', label: 'Leave Team dialog'},
-            {id: 'integrations', text: 'Integrations'},
-            {id: 'marketplaceModal', label: 'Marketplace dialog'},
-            {id: 'systemConsole', text: 'System Console'},
-            {id: 'helpLink', text: 'Help', directLink: true},
-            {id: 'gettingStarted', text: 'Getting Started'},
-            {id: 'keyboardShortcuts', text: 'Keyboard Shortcuts'},
-            {id: 'reportLink', text: 'Report a Problem', directLink: true},
-            {id: 'nativeAppLink', text: 'Download Apps', directLink: true},
-            {id: 'about', label: `About ${siteName} dialog`},
-            {id: 'logout', text: 'Log Out'},
```

Comment:
Other items will be fixed separately once we have final list of where the items will go to what menus. If anyone has info, then will use it as reference.